src/server/src/organizations/services/organizations.service.ts

```diff
@@ -12,6 +13,46 @@ export class OrganizationsService extends TypeOrmCrudService<Organization> {
   }
 
   save(org: Organization): Promise<Organization> {
+    // @ts-ignore
     return this.repo.save(org);
   }
+
+  async getOrgAndProjects(slug: string): Promise<Organization | undefined> {
+    const builder = this.repo.createQueryBuilder("organization");
+    const data = await builder
+      .innerJoinAndSelect("organization.users", "users")
+      .innerJoinAndSelect("organization.admin", "admin")
```

Comment:
These should both be `leftJoinAndSelect` as well.
```suggestion
      .leftJoinAndSelect("organization.users", "users")
      .leftJoinAndSelect("organization.admin", "admin")
```